pkg/interop/native/gas/gas.go

```diff
@@ -31,5 +31,5 @@ func BalanceOf(addr interop.Hash160) int {
 // Transfer represents `transfer` method of GAS native contract.
 func Transfer(from, to interop.Hash160, amount int, data interface{}) bool {
 	return contract.Call(interop.Hash160(Hash), "transfer",
-		contract.WriteStates|contract.AllowCall|contract.AllowNotify, from, to, amount, data).(bool)
+		contract.All, from, to, amount, data).(bool)
```

Comment:
And we also have NameService `Transfer` interop.

Reply:
And it doesn't have `AllowCall`, ouch.

Reply:
Though that's a separate issue. Fixed reads.